.travis.yml

```diff
@@ -62,7 +62,6 @@ before_install:
 install:
   - travis_wait composer global update --prefer-dist --no-interaction
   - travis_wait composer update --prefer-dist --no-interaction --prefer-stable $COMPOSER_FLAGS
-  - pip install -r Resources/doc/requirements.txt --user `whoami`
```

Comment:
requirements.txt does not only add Sphinx: https://github.com/sonata-project/SonataAdminBundle/blob/5dc9071790d0889a2de7482869e0b381c52b7d15/Resources/doc/requirements.txt